src/test/scala/com/atomist/rug/RugPackagingTest.scala

```diff
@@ -0,0 +1,61 @@
+package com.atomist.rug
```

Comment:
We need these tests right now, especially during the refactoring work I've been doing.